docs/cli.rst

```diff
@@ -0,0 +1,45 @@
+Command Line Interface
+======================
+
+Nameko ships with a command line interface to make hosting and interacting with services as easy as possible.
+
+Running a Service
+-----------------
+
+::
+
+    $ nameko run [<module>[:<ServiceClass>]]
```

Comment:
Too late maybe more `module.Class` is more common. https://docs.djangoproject.com/en/1.7/ref/settings/